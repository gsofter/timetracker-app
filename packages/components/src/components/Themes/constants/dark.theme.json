{
	"name": "dark",
	"type": "dark",
	"colors": {
		"primaryColor": "#1890ff",
		"app.background": "#fff",
		"app.headerHeight": "48px",
		"list.dropBackground": "#383b3d",
		"sideBarTitle.foreground": "#bbbbbb",
		"popover.background": "#1f1f1f",
		"popover.customize.border.color": "#3a3a3a",
		"body.background": "#000000",
		"component.background": "#141414",
		"layout.body.background": "#000000",
		"layout.header.background": "#1f1f1f",
		"layout.trigger.background": "#262626"
	},
	"tokenColors": [
		{
			"scope": "header",
			"settings": {
				"foreground": "#000080"
			}
		},
		{
			"scope": "text",
			"settings": {
				"foreground": "rgba(255,255,255,0.85)"
			}
		},
		{
			"scope": "text.secondary",
			"settings": {
				"foreground": "rgba(255,255,255,0.65)"
			}
		}
	]
}