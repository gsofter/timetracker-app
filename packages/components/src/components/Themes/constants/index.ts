export * from './actionTypes';
export { default as COLORS } from './colors';
export { default as THEMES } from './themes';
