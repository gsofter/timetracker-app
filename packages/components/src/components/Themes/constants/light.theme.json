{	
	"name": "light",
	"type": "light",
	"colors": {
		"primaryColor": "#1890ff",
		"app.background": "#000",
		"app.headerHeight": "48px",
		"list.dropBackground": "#383b3d",
		"sideBarTitle.foreground": "#bbbbbb",
		"popover.background": "#1f1f1f",
		"popover.customize.border.color": "#3a3a3a",
		"body.background": "#000000",
		"component.background": "#141414",
		"layout.body.background": "#000000",
		"layout.header.background": "#1f1f1f",
		"layout.trigger.background": "#262626"
	},
	"tokenColors": [
		{
			"scope": "header",
			"settings": {
				"foreground": "#262626"
			}
		},
		{
			"scope": "text",
			"settings": {
				"foreground": "#595959"
			}
		},
		{
			"scope": "text.secondary",
			"settings": {
				"foreground": "#8c8c8c"
			}
		}
	]
}