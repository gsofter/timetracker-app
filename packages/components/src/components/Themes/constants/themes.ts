import { IThemeSettings } from '../interfaces';
const lightSettings: IThemeSettings = require('./light.theme.json');
const darkSettings: IThemeSettings = require('./dark.theme.json');
const hcSettings: IThemeSettings = require('./hc.theme.json');

export default [
  {
    name: 'light',
    className: '',
    text: 'Light',
    defaultSettings: lightSettings,
  },
  {
    name: 'dark',
    className: 'vs-dark',
    text: 'Dark',
    defaultSettings: darkSettings,
  },
  {
    name: 'hc',
    className: 'hc-black',
    text: 'High Contrast',
    defaultSettings: hcSettings,
  },
];
