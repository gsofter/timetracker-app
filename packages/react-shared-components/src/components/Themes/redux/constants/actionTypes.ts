export const THEME_ACTION_TYPES = {
    SET_VALUE: 'theme/SET_VALUE',
    BULK_UPDATE: 'theme/BULK_UPDATE',
    LOAD_THEME: 'theme/LOAD_THEME',
    SET_THEME: 'theme/SET_THEME',
    SET_SETTING: 'theme/SET_SETTING',
};
