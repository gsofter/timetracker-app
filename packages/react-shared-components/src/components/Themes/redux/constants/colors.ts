export default [
  {
    name: 'Black',
    light: '#000000',
    dark: '#000000',
    hc: '#000000',
  },
  {
    name: 'Red',
    light: '#cd3131',
    dark: '#cd3131',
    hc: '#cd0000',
  },
  {
    name: 'Green',
    light: '#00BC00',
    dark: '#0DBC79',
    hc: '#00cd00',
  },
  {
    name: 'Yellow',
    light: '#949800',
    dark: '#e5e510',
    hc: '#cdcd00',
  },
  {
    name: 'Blue',
    light: '#0451a5',
    dark: '#2472c8',
    hc: '#0000ee',
  },
  {
    name: 'Black',
    light: '#bc05bc',
    dark: '#bc3fbc',
    hc: '#cd00cd',
  },
  {
    name: 'Cyan',
    light: '#0598bc',
    dark: '#11a8cd',
    hc: '#00cdcd',
  },
  {
    name: 'White',
    light: '#555555',
    dark: '#e5e5e5',
    hc: '#e5e5e5',
  },
  {
    name: 'BrightBlack',
    light: '#666666',
    dark: '#666666',
    hc: '#7f7f7f',
  },
  {
    name: 'BrightRed',
    light: '#cd3131',
    dark: '#f14c4c',
    hc: '#ff0000',
  },
  {
    name: 'BrightGreen',
    light: '#14CE14',
    dark: '#23d18b',
    hc: '#00ff00',
  },
  {
    name: 'BrightYellow',
    light: '#b5ba00',
    dark: '#f5f543',
    hc: '#ffff00',
  },
  {
    name: 'BrightBlue',
    light: '#0451a5',
    dark: '#3b8eea',
    hc: '#5c5cff',
  },
  {
    name: 'BrightMagenta',
    light: '#bc05bc',
    dark: '#d670d6',
    hc: '#ff00ff',
  },
  {
    name: 'BrightCyan',
    light: '#0598bc',
    dark: '#29b8db',
    hc: '#00ffff',
  },
  {
    name: 'BrightWhite',
    light: '#a5a5a5',
    dark: '#e5e5e5',
    hc: '#ffffff',
  },
];
