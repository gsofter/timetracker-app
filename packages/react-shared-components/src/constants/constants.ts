


export const CHANGE_SETTINGS_ACTION = 'settings/changeSetting';