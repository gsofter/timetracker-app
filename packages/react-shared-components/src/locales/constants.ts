export const LOCALES = {
    EN_US: 'en-US',
    PT_BR: 'pt-BR',
    ZH_CN: 'zh-CN',
    ZH_TW: 'zh-TW',
    ID_ID: 'id-ID',
  };