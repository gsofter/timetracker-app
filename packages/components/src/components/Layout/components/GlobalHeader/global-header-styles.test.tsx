import 'raf/polyfill'
import * as React from 'react';
import { createSnapshot } from 'jest-react-fela';
import { createComponent } from 'react-fela';
import { THEMES } from '../../../Themes/redux/constants/index'

import { styleSheet } from './styles-new';
import 'jest';


const rule = (props) => styleSheet.globalHeaderStyle({ 
  theme: THEMES[0], 
  prefixCls: 'ant-pro' }) as any;
describe('Test styles', () => {

  it('should return a Component', () => {


    const Component = createComponent(rule)

    expect(createSnapshot(<Component className='ant-pro' />)).toMatchSnapshot()
  })

})


