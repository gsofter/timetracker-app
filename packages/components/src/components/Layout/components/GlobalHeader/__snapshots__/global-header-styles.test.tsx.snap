// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test styles should return a Component 1`] = `
".a .ant-pro-global-header {
  position: relative;
}
.b .ant-pro-global-header {
  display: flex;
}
.c .ant-pro-global-header {
  align-items: center;
}
.d .ant-pro-global-header {
  height: 100%;
}
.e .ant-pro-global-header {
  padding: 0 16px;
}
.f .ant-pro-global-header {
  background: #fff;
}
.g .ant-pro-global-header {
  box-shadow: 0 1px 4px rgba(0, 21, 41, 0.08);
}
.h .ant-pro-global-header > * {
  height: 100%;
}
.i .ant-pro-global-header-collapsed-button {
  display: flex;
}
.j .ant-pro-global-header-collapsed-button {
  align-items: center;
}
.k .ant-pro-global-header-collapsed-button {
  margin-left: 16px;
}
.l .ant-pro-global-header-collapsed-button {
  font-size: 20px;
}
.m .ant-pro-global-header-layout-mix {
  background-color: #001529;
}
.n .ant-pro-global-header-layout-mix .anticon {
  color: #1890ff;
}
.o .ant-pro-global-header-logo {
  position: relative;
}
.p .ant-pro-global-header-logo {
  overflow: hidden;
}
.q .ant-pro-global-header-logo a {
  display: flex;
}
.r .ant-pro-global-header-logo a {
  align-items: center;
}
.s .ant-pro-global-header-logo a {
  height: 100%;
}
.t .ant-pro-global-header-logo a img {
  height: 28px;
}
.u .ant-pro-global-header-logo a h1 {
  height: 32px;
}
.v .ant-pro-global-header-logo a h1 {
  margin: 0 0 0 12px;
}
.w .ant-pro-global-header-logo a h1 {
  color: #1890ff;
}
.x .ant-pro-global-header-logo a h1 {
  font-weight: 600;
}
.y .ant-pro-global-header-logo a h1 {
  font-size: 18px;
}
.z .ant-pro-global-header-logo a h1 {
  line-height: 32px;
}
.ab .ant-pro-global-header-menu .anticon {
  margin-right: 8px;
}
.ac .ant-pro-global-header-menu .ant-dropdown-menu-item {
  min-width: 160px;
}
.ae .ant-pro-global-header .dark {
  height: 48px;
}
.af .ant-pro-global-header .dark .action {
  color: rgba(255, 255, 255, 0.85);
}
.ag .ant-pro-global-header .dark .action > i {
  color: rgba(255, 255, 255, 0.85);
}
.ah .ant-pro-global-header .dark .action:hover {
  background: #1890ff;
}
.ai .ant-pro-global-header .dark .action.opened {
  background: #1890ff;
}
.aj .ant-pro-global-header .dark .action .ant-badge {
  color: rgba(255, 255, 255, 0.85);
}


<div className=ant-pro a b c d e f g h i j k l m n o p q r s t u v w x y z ab ac ae af ag ah ai aj />;
"
`;
